public/app/features/explore/Wrapper.test.tsx

```diff
@@ -64,7 +64,7 @@ describe('Wrapper', () => {
     (datasources.loki.query as Mock).mockReturnValueOnce(makeLogsQueryResponse());
 
     // Make sure we render the logs panel
-    await screen.findByText(/^Logs$/i);
```

Comment:
As we have in Older logs button the word logs, I needed to remove case insensivity to make it work